server/mvc/models/ThinkyDocumentModel.js

```diff
@@ -24,7 +24,8 @@ var ThinkyDocumentModel = inherit(ConfigModel, {
         .error(reject);
     });
   },
-  findByFilter: function(filter, orderBy){
+  findByFilter: function(filter, orderBy, num){
+    num = typeof num !== "undefined" ? num : 1;
```

Comment:
This model structure is only really geared for matching a single document. Ideally, we'd have a model abstraction split up into "single" and "collection", where collections may contain one-or-more documents. In this case, we'd then use a collection.